### DISCUSSION TITLE
Struggling to implement an external converter, it's not used.

### DISCUSSION THREAD

**Discussion by PSTB** [Koenkk/zigbee2mqtt#16125] [Devices]
I'm trying to add more support for device Wiser/Schneider CCTFR6400 (already partly supported) by using an external converter.

My external converter file seems basically to be OK (no errors logged, zigbee2mqtt starts correctly when it's defined in the configuration.yaml), however when pairing the device, from what I can see my external converter is never used.

I understood that external converters have predence over existing internal converters.
Is this still the case ? Any tips & tricks to know about ?
I'm using Z2M 1.28.4 commit 52e545

Thanks for any help.

**Comment by rmazzali** [comment]
Hello it also happens to me: did you fixed it? if yes can you explain how? thank you
